src/main/java/org/graylog/aws/inputs/cloudtrail/CloudTrailSubscriber.java

```diff
@@ -155,6 +160,7 @@ public void run() {
                             }
 
                             sourceInput.processRawMessage(new RawMessage(objectMapper.writeValueAsBytes(record)));
+                            sourceInput.stop();
```

Comment:
This can't be right 🤔

Reply:
oops. how did that get here :grimacing: thanks!